requirements-dev.txt

```diff
@@ -21,7 +21,7 @@ beautifulsoup4==4.6.0
 # The below pins mirror the Python version-conditional pins in setup.py
 #
 Jinja2>=2.10.1; python_version >= '3.6'
-mock; python_version >= '3.6'
+mock<=4.0.2; python_version >= '3.6'
```

Comment:
Separate PR is outstanding to make it work with 4.0.3, this is just to get the build to pass.